apps/admin/backend/src/app.results_csv_export.test.ts

```diff
@@ -194,114 +160,6 @@ it('logs failure if export fails for some reason', async () => {
   );
 });
 
-it('uses appropriate file headers based on group by', async () => {
```

Comment:
All the testing removed from this file was reproduced in `./exports/csv_results.test.ts`.